pkgs/applications/science/astronomy/stellarium/default.nix

```diff
@@ -1,20 +1,22 @@
 { stdenv, fetchurl, cmake, freetype, libpng, mesa, gettext, openssl, perl, libiconv
 , qtscript, qtserialport, qttools, makeQtWrapper
+, qtmultimedia
 }:
 
 stdenv.mkDerivation rec {
-  name = "stellarium-0.14.3";
+  name = "stellarium";
```

Comment:
Why remove the version number from the name? See the [package naming](https://nixos.org/nixpkgs/manual/#sec-package-naming) section in the manual.